easybuild/easyconfigs/a/ASE/ASE-3.21.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.21.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-%(pyver)s'),
+    ('Flask', '1.1.2', '-Python-%(pyver)s', ('GCCcore', '9.3.0')),
+    ('matplotlib', '3.2.1', '-Python-%(pyver)s'),
+    ('Tkinter', '%(pyver)s', '', ('GCCcore', '9.3.0')),  # needed by GUI of ASE
```

Comment:
Again, I'm not sure the `GCCcore` part is needed here.

Reply:
Because it depends on `Tkinter-3.8.2-GCCcore-9.3.0.eb`, I would like to include this explicitly to avoid ambiguity.

Reply:
For both `Flask` and `Tkinter`, the corresponding easyconfigs using `GCCcore` will be considered automatically by EasyBuild when `eb --robot` is used.

Hardcoding the version of the subtoolchain here will make it more difficult to update this easyconfig file later: if the `toolchain` value is tweaked, then you'd have to change the hardcoded `GCCcore` versions here too.

We don't do this anywhere in the central easyconfigs repository, so please remove the hardcoding:

```python
dependencies = [
    ('Python', '3.8.2'),
    ('SciPy-bundle', '2020.03', versionsuffix),
    ('Flask', '1.1.2', versionsuffix),
    ('matplotlib', '3.2.1', versionsuffix),
    ('Tkinter', '%(pyver)s'),  # needed by GUI of ASE
    ('spglib-python', '1.16.0', versionsuffix),  # optional
]

Reply:
Thanks for the info! done and passed the checks.